recipes/karilang-kernel/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "karilang-kernel" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Vipul-Cariappa/KariLang-Kernel/releases/download/v0.1.0/KariLang-Kernel-withsubmodules-0.1.0.tar.gz
+  sha256: a86cf4d31ae77a88dfb0c639f028fd9261bd890b128469553c38fecaf31bf33c
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bison  # [unix]
+    - flex  # [unix]
```

For Windows there are a couple options. Either use MSYS2 packages

```suggestion
    - bison     # [unix]
    - flex      # [unix]
    - m2-bison  # [win]
    - m2-flex   # [win]
```

Or use WinFlexBison

```suggestion
    - bison         # [unix]
    - flex          # [unix]
    - winflexbison  # [win]
```

---

I followed your suggestion to use `m2-bison` and `m2-flex` from MSYS2 and `winflexbison` also. Both of them add the `unistd.h` as a required header file in the `.c` file they output. Which cannot be compiled using MSVC (Visual Studio) on Windows because MSVC does not provide that header file. Any suggestion on how I can proceed? Is there a way to change the compiler to use MSYS2's `c/c++` compiler?
Thanks for the suggestions so far.

---

I managed to fix it by defining `YY_NO_UNISTD_H`.
Thanks for the help again.